install/01_clusterautoscaler.crd.yaml

```diff
@@ -17,7 +17,6 @@ spec:
     - ca
     singular: clusterautoscaler
   scope: Cluster
-  preserveUnknownFields: false
```

Comment:
Is this required?  @enxebre I remember you introduced `preserveUnknownFields: false` intentionally

Reply:
I see the comment below now